R/timeseries_plot.R

```diff
@@ -0,0 +1,100 @@
+# multiple stations (month)
+niger_data <- readRDS("C:/Users/lzc1n17/OneDrive - University of Southampton/PhD/IDEMS/RInstatClimatic/Niger4_day_month_year.RDS")
+niger_data <- niger_data$get_data_frame()
+niger_data_1 <- niger_data$Data
+niger_monthly_data <- niger_data$Data_by_year_month
+niger_yearly_data <- niger_data$Data_by_year
+niger_monthly_data$station = ifelse(niger_monthly_data$month_abbr %in% c("Jan", "Feb", "Mar"), "A", "B")
```

Comment:
Could you do testing in a separate file? Then it won't show up in the commit history.